packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -241,6 +270,11 @@ export default defineComponent({
         ? !state.selectedRange.start || !state.selectedRange.end
         : !state.selectedRange.start
     })
+    const widthStyle = computed(() => {
+      return {
+        width: props.width === 'auto' || props.width.endsWith('%') || props.width.endsWith('px') ? props.width : props.width + 'px'
+      }
+    })
```

Comment:
We have a utility function that should handle this for you. Here's what you'll need (please test though)

```ts
import useUtilities from '@/composables/useUtilities'

const { getSizeFromString } = useUtilities()

const widthStyle = computed(() => {
  return getSizeFromString(props.width)
})
```

Reply:
Only available on beta branch :)

But yes, I will make sure to use this on my _other_ branch 👍

Reply:
oops 😢